tests/Doctrine/Tests/ORM/Functional/Ticket/GH9027Test.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\UnitOfWork;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use Exception;
+
+class GH9027Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        try {
+            $this->_schemaTool->createSchema(
+                [
+                    $this->_em->getClassMetadata(GH9027Customer::class),
+                    $this->_em->getClassMetadata(GH9027Cart::class),
+                ]
+            );
+        } catch (Exception $e) {
+        }
```

Comment:
Can you please use something similar to what was done in https://github.com/doctrine/orm/pull/8962 ?

Reply:
Done, sorry for using an old template 🤦🏽

Reply:
No worries, really, I would have done the same :)